cmd/program/program.go

```diff
@@ -317,11 +309,20 @@ func (p *Project) CreateMainFile() error {
 
 	defer makeFile.Close()
 
-	// inject makefile template
-	makeFileTemplate := template.Must(template.New("makefile").Parse(string(framework.MakeTemplate())))
-	err = makeFileTemplate.Execute(makeFile, p)
-	if err != nil {
-		return err
+	if p.DBDriver == "sqlite" || p.DBDriver == "none" {
+		// inject makefile template
+		makeFileTemplate := template.Must(template.New("makefile").Parse(string(framework.NonDbMakeFileTemplate())))
+		err = makeFileTemplate.Execute(makeFile, p)
+		if err != nil {
+			return err
+		}
+	} else {
+		// inject makefile template
```

Comment:
nit: Maybe change this to say this one includes the commands for Databases as well